.github/workflows/reusable-git-metadata-test.yml

```diff
@@ -87,12 +62,7 @@ jobs:
       - name: Execute git-metadata-test
         run: |-
           ${{inputs.SUDO}} ${{ inputs.BUILT_EARTHLY_PATH }} --ci -P \
-            --build-arg DOCKERHUB_AUTH=true \
-            --build-arg DOCKERHUB_USER_SECRET=+secrets/earthly-technologies/dockerhub-mirror/user \
-            --build-arg DOCKERHUB_TOKEN_SECRET=+secrets/earthly-technologies/dockerhub-mirror/pass \
-            --build-arg DOCKERHUB_MIRROR=registry-1.docker.io.mirror.corp.earthly.dev \
           ./tests/git-metadata+test
-        if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name == github.repository
```

Comment:
I know this isn't the first deletion of this condition but just wondering where it moved to, if anywhere.

Reply:
It was unnecessary -- because outside contributors' PRs need to be approved before they are run -- it's up to us to look over the code to prevent malicious PRs.